migration-guides/v6.0.0.md

```diff
@@ -193,3 +194,24 @@ const PreviewComponent = ({ file }) => {
 }
 <FileInput previewComponent={PreviewComponent} />
 ```
+
+#### 9. The tag on `<Spinner />` now uses the classname `spinner` in place of an id
+
+`<Spinner />` utilizes a classname and can accept additional custom classes in order to accommodate views where more than one spinner is necessary.
+
+```scss
+// Before
+#spinner {
+  text-align: center;
+}
+
+// After
+.spinner {
+  text-align: center;
+}
+
+// After
+.custom-spinner {
+  text-align: center;
```

Comment:
It'd be great if you chose another declaration here to really showcase the benefit of customizing just a specific spinner. When I read this example, you're not changing anything from above so it reads a little confusing to me. E.g., 
```css
.custom-spinner {
  margin-bottom: 10px;
}